contracts/test/mocks/MockGateway.sol

```diff
@@ -180,27 +210,27 @@ contract MockGateway is IAxelarGateway {
         return address(0);
     }
 
-    function tokenMintLimit(string memory symbol) public view override returns (uint256) {
```

Comment:
Simplify the mock gateway by removing all these unused methods

Reply:
You can see the methods on MockGateway that are used in tests and just keep those only (feel free to simplify the method logic, e.g. deployToken can just deploy the token but not worry about mint limit etc. GMP test can be simplified if needed

Reply:
yup, make sense. updated.